File: app/src/main/java/com/example/narthana/genpass/preference/Utility.java
package com.example.narthana.genpass.preference;

import android.content.Context;
import android.util.TypedValue;

/**
 * Created by narthana on 26/01/17.
 */

class Utility
{
    static int dpToPx(Context context, int length)
    {
        return (int) TypedValue.applyDimension(
                TypedValue.COMPLEX_UNIT_DIP,
                length,
                context.getResources().getDisplayMetrics()
        );
    }
}

File: app/src/main/java/com/example/narthana/genpass/preference/Utility.kt
package com.example.narthana.genpass.preference

import android.content.Context
import android.util.TypedValue

/**
 * Created by narthana on 26/01/17.
 */

internal object Utility {
    fun dpToPx(context: Context, length: Float) = TypedValue.applyDimension(
        TypedValue.COMPLEX_UNIT_DIP,
        length,
        context.resources.displayMetrics
    ).toInt()
}